assets/dev/js/frontend/handlers/container/container.js

```diff
@@ -3,5 +3,9 @@ export default [
 	() => import( /* webpackChunkName: 'container' */ '../background-video' ),
 	() => import( /* webpackChunkName: 'container' */ './handles-position' ),
 	// TODO: Move to a global handler for all widgets.
-	() => import( /* webpackChunkName: 'container' */ '../floating-bar' ),
+	() => {
+		if ( elementorFrontend.isEditMode() ) {
+			return import( /* webpackChunkName: 'container' */ '../floating-bar' );
+		}
+	},
```

Comment:
@kobizz

Reply:
1. Although this handler will run only in edit mode, its code will be included in the `container` chunk that runs in the frontend.
2. Because you use here a different chunk, it seems that all the above handlers (`background-slideshow, background-video, handles-position`) will be duplicated in the frontend, since they are already loaded under `assets/dev/js/frontend/handlers/section/section.js`.

Reply:
Talked with @rotemee, see this commit:
https://github.com/elementor/elementor/pull/16009/commits/c5a5615ba407403eaf38c8a5ab4b711e7c0c4c71